src/_gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -314,14 +318,14 @@ def eink_st_sonderausgaben_y_tu_mit_betreuung(
 
 def eink_st_abz_betreuungskost_y(
     eink_st_abzuege_params: dict,
-    betreuungskost_m: float,
+    betreuungskost_eltern_m: float,
```

Comment:
```suggestion
    betreuungskost_eltern_y: float,
```

Although I think I'd prefer `betreuungskost_elternteil_y`, if I understand the variable correctly.